File: Android-app/domain/src/main/java/uj/roomme/domain/schedule/ScheduleModel.kt
package uj.roomme.domain.schedule

import uj.roomme.domain.housework.HouseworkSettingsModel
import uj.roomme.domain.housework.HouseworkShortModel
import uj.roomme.domain.housework.HouseworkStatusModel
import uj.roomme.domain.user.UserNicknameModel
import java.time.OffsetDateTime

data class ScheduleModel(
    val id: Int,
    val housework: HouseworkShortModel,
    val user: UserNicknameModel,
    val date: OffsetDateTime,
    val status: HouseworkStatusModel,
    val settings: HouseworkSettingsModel
)

File: Backend/RoomMe/RoomMe.API/Models/ScheduleModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ScheduleModel
    {
        public int Id { get; set; }
        public HouseworkShortModel Housework { get; set; }
        public UserNicknameModel User { get; set; }
        public DateTime Date { get; set; }
        public HouseworkStatusModel Status { get; set; }
    }
}
